style.css

```diff
@@ -1,8 +1,98 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+@import url("https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@600&family=Oswald&display=swap");
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+:root {
+  /* FONT */
+  --font-family-heading: "Nunito Sans", sans-serif;
+  --font-family-text: "Oswald", sans-serif;
+
+  /* COLORS */
+  --color-heading: #202329;
+  --color-text: #333;
+  --color-contrast: #fff;
+  --color-hover: #ff9800;
+  --background-color-main: #24282f;
+  --background-color-footer: #202329;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: var(--font-family-heading);
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  font-family: var(--font-family-text);
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 1.2em;
+  margin: 1em 0 1em 0;
+}
+
+header {
```

Comment:
Be careful with element selectors here. 
`header` might be used in other components too.